config/deploy.rb

```diff
@@ -0,0 +1,100 @@
+require 'mina/rails'
+require 'mina/git'
+require 'mina/rvm'
+require 'mina/puma'
+# install_plugin Mina::Puma
+# require 'mina/rbenv'  # for rbenv support. (https://rbenv.org)
+# require 'mina/rvm'    # for rvm support. (https://rvm.io)
+
+# Basic settings:
+#   domain       - The hostname to SSH to.
+#   deploy_to    - Path to deploy into.
+#   repository   - Git repo to clone from. (needed by mina/git)
+#   branch       - Branch name to deploy. (needed by mina/git)
+
+set :application_name, 'VM-Managment Portal'
+set :domain, 'vm-swt-hrmt-master.eaalab.hpi.uni-potsdam.de'
+set :deploy_to, '/var/www/vm-swt-hrmt-master.eaalab.hpi.uni-potsdam.de'
+set :repository, 'git@github.com:hpi-swt2/vm-portal.git'
+set :branch, 'deploy'
```

Comment:
Change this to `master` and think about multistaging

Reply:
There is a mina multi-staging gem! We can discuss this after we copied the VM.